### PR TITLE
Add support for LLMS and SEMANTIC_INDEXES resources

### LINKED ISSUE #20: Update for LLM work

### PR DESCRIPTION
**Pull request by sfitts**
Removed explicit configuration of the logger in the library as this violates a Python convention (this is something that the application using the library should do).

Also update the build/configure to bring it into alignment with our other Python projects (basically modernize it a bit).

Fixes #20

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -55,29 +69,19 @@ python {
...
     // used virtualenv path (if virtualenv used, see 'scope')
-    envPath
+    if (project.hasProperty('virtualenvPath')) {
+        envPath = virtualenvPath
+    }
     // copy virtualenv instead of symlink (when created)
     envCopy = false
+}
 
-    pip 'pip:23.1.2'
```
Comment (sfitts) [inline build.gradle:62]: Done via pip-tools now.

File: pyproject.toml
Comment (sfitts) [inline pyproject.toml:1]: Generated via a template (instead of generating the setup config, since that's an older approach).

**Review by skyler1ackerman — APPROVED** [review]
LGTM

**Review by fhcarter — APPROVED** [review]
LGTM

**Pull request merged**